modules/lti/src/main/resources/tools/deeplink/deeplink.js

```diff
@@ -246,36 +364,96 @@ function populateData(title, image, created, tool) {
     $('#test').val(urlParams.get('test'));
   }
 
+  // retrieve start & end
+  if (range) {
+    start =  parseInt(range[0]);
+    end = parseInt(range[1]);
+    customData = `?custom_start=${start}&custom_end=${end}`;
+  }
+
   // generate content_items
   let contentItems = {
     '@context': 'http://purl.imsglobal.org/ctx/lti/v1/ContentItem',
     '@graph': [{
       '@type': 'LtiLinkItem',
+      '@id': mpID,
       mediaType: 'application/vnd.ims.lti.v1.ltilink',
-      title: title,
-      text: created,
-      thumbnail: {'@id': image},
+      title: data.title,
+      text: data.created,
+      thumbnail: {
+        '@id': data.image,
+        'height': '400',
+        'width': '700'
+      },
       custom: {
-        tool: tool
+        tool: data.tool,
+        start: start,
+        end: end
```

Comment:
Canvas ignores the custom tool data, but leaving it in for future compatibility.